commandLine/src/addons/ofAddon.cpp

```diff
@@ -772,65 +801,148 @@ void ofAddon::parseLibsPath(const fs::path & libsPath, const fs::path & parentFo
 			; // do we need to do anything here?
 		} else {
 			// if addon is local, it is relative to the project folder, and if it is not, it is related to the project folder, ex: addons/ofxSvg
-			fs::path rel = fs::relative (f, isLocalAddon ? pathToProject : pathToOF);
-			fs::path folder = rel.parent_path();
-
-			if (isLocalAddon) {
-				fs::path fFS { f };
-				folder = fs::path { "local_addons" } / fs::relative(fFS.parent_path(), parentFolder);
-			}
-
-			filesToFolders[f] = folder;
+//			fs::path rel = fs::relative (f, isLocalAddon ? pathToProject : pathToOF);
+//			fs::path folder = rel.parent_path();
+//
+//			if (isLocalAddon) {
+//				fs::path fFS { f };
+//				folder = fs::path { "local_addons" } / fs::relative(fFS.parent_path(), parentFolder);
+//			}
+//
+//			filesToFolders[f] = folder;
+            addToFolder(f, parentFolder);
 		}
 	}
 
 	for (const auto & f : xcframeworks) {
 		
-		fs::path rel = fs::relative(f, isLocalAddon ? pathToProject : pathToOF);
-		fs::path folder = rel.parent_path();
-
-		if (isLocalAddon) {
-			fs::path fFS { f };
-			folder = fs::path { "local_addons" } / fs::relative(fFS.parent_path(), parentFolder);
-		}
-
-		filesToFolders[f] = folder;
+//		fs::path rel = fs::relative(f, isLocalAddon ? pathToProject : pathToOF);
+//		fs::path folder = rel.parent_path();
+//
+//		if (isLocalAddon) {
+//			fs::path fFS { f };
+//			folder = fs::path { "local_addons" } / fs::relative(fFS.parent_path(), parentFolder);
+//		}
+//
+//		filesToFolders[f] = folder;
+        addToFolder(f, parentFolder);
 	}
 }
+string ofAddon::cleanName(const string& name){
+    auto addonName = name;
+#ifdef TARGET_WIN32
```

Comment:
this define is wrong, so if not addressed now at least keep the comment
// FIXME : not target, yes platform.
needed if somebody is generating windows projects in a linux/macos machine

Reply:
I just took that from what was there before.

Reply:
yes, just keep the comment and it can be addressed soon